deploy-replay-job

```diff
@@ -71,14 +71,16 @@ if ! [[ "$END_TIME" =~ ^$VALID_TIME ]]; then
 fi
 
 case $PROFILE in
-  staging) AWS_REGION="eu-central-1"
+  staging) KINESIS_REGION="eu-central-1"
            KINESIS_ENDPOINT="kinesis.eu-central-1.amazonaws.com"
            S3_BASE_DIR="staging-witan-event-log"
+           S3_REGION="eu-central-1"
            METRONOME_URL="http://masters.staging.witan.mastodonc.net/service/metronome/v1/"
            ;;
-  prod) AWS_REGION="eu-west-1"
-        KINESIS_ENDPOINT="kinesis.eu-west-1.amazonaws.com"
+  prod) KINESIS_REGION="eu-central-1"
```

Comment:
Did `prod` intend to change from `eu-west-1` to `eu-central-1`?

Reply:
Yeah, well picked up on though.

The intention here is that in 'prod' mode it loads from prod into staging. So the S3 bucket is in eu-west-1, but the kinesis stream is in central.

I figure this will be the much more 'normal' way things go, but all the levers are there for doing a prod to prod load should it be needed.